packages/builder/src/templates/generic/useHttpMutation.hbs

```diff
@@ -50,7 +50,7 @@ const useHttpMutation = <Req, Res>(params: UseMutationParams<Res>) => {
       });
       const contentType = response.headers.get("Content-Type");
       if (response.ok) {
-        if (contentType === 'application/json') return response.json() as Promise<Res>;
+        if (contentType?.includes('application/json')) return response.json() as Promise<Res>;
```

Comment:
Can we add a test case for this?
I think we have test here `packages/react-app-example/src/__test__/useHttpMutation.spec.tsx`